### ISSUE TITLE
index error after saving labels and going to next frame

### ISSUE THREAD

**Issue by backyardbiomech** [DeepLabCut/DeepLabCut#1110]
**Desktop (please complete the following information about your system):**
 - OS: macOS 11.2
 - DeepLabCut Version: 2.1.9, 2.1.10.1
 - DeepLabCut mode: single animal
 - Browser, if applicable [e.g. chrome, safari]

**To Reproduce**

1. Create project, add video, extract frames.
2. Open label frames GUI, and label first frame.
3. Click Save
4. Click Next
4. See error:

<details><summary>TRACEBACK</summary><p>

```python

pythonw -m deeplabcut
Starting GUI...
Found new frames..
Traceback (most recent call last):
  File "/Users/brandonjackson/opt/miniconda3/envs/dlc/lib/python3.7/site-packages/deeplabcut/generate_training_dataset/labeling_toolbox.py", line 767, in nextImage
    self.updatedCoords = MainFrame.getLabels(self, self.iter)
  File "/Users/brandonjackson/opt/miniconda3/envs/dlc/lib/python3.7/site-packages/deeplabcut/generate_training_dataset/labeling_toolbox.py", line 845, in getLabels
    self.dataFrame[self.scorer][bp]["x"].values[self.iter],
IndexError: index 1 is out of bounds for axis 0 with size 1

```
</p></details>

**Additional context**

Discussed on the [forum](https://forum.image.sc/t/label-frames-dataframe-index-out-of-bounds-error/48653/4).

The `Found new frames..` message only prints after the save, upon clicking Next. 
The error does not occur on these installations when just labelling and then clicking next. The saved data are complete for the labeled frames, but unlabeled (i.e. not viewed) frame paths are not saved into `CollectedData...csv`.

Obviously, saving is not absolutely necessary after each frame, but when applying 30 labels per image it might provide some peace-of-mind. Otherwise, "Save" should be "Save & Quit".

**Comment by MMathisLab** [comment]
yeah, this is due to an unforeseen bug that was introduced with the delete button that we did not catch. #1116 fixes this bug! sorry for this trouble, we will issue a new DLC 2.1.10.2 today for this. I hit it last night as well and caused some major pain ;(.

**Comment by stephmshields** [comment]
Has this been fixed for 2.2b8 as well? I'm working with some new undergrads (using MacOS & Ubuntu), and they've encountered the same problem, which my undergrad in the fall (using MacOS) didn't report having. I can get more details from them about their operating systems, etc. if necessary!

**Comment by MMathisLab** [comment]
Beta 8 is no longer a working version. Please use 2.10.2; which has all the beta features and more ;) -- yes, solved!
